setup-yarn/action.yml

```
@@ -26,6 +26,7 @@ runs:
           node_modules
           frontend/*/node_modules
           */node_modules
+          code/*/node_modules
```

Globstar is supported (https://github.com/actions/cache)
>path - A list of files, directories, and wildcard patterns to cache and restore. See [@actions/glob](https://github.com/actions/toolkit/tree/main/packages/glob) for supported patterns.

So we should be able to do:
```suggestion
          **/node_modules
```